bentoml/saved_bundle/config.py

```diff
@@ -206,6 +208,12 @@ def get_bento_service_metadata_pb(self):
                     else:
                         api_metadata.mb_max_batch_size = DEFAULT_MAX_BATCH_SIZE
 
+                    # Supports customize route from 0.10.2
```

Comment:
Remove this comment?  the next release will be 0.11.0

Reply:
done